### PR TITLE
User can upvote/downvote reviews

### PR DESCRIPTION
**Pull request by sarah-janewilson**

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -23,4 +23,14 @@ export const fetchCommentsByReviewId = (review_id) => {
...
+};
+
+export const voteOnSingleReview = (review_id, increment) => {
+  const patchVotes = {
+    inc_votes: increment ? 1 : -1,
+  };
+  return gamesApi
+    .patch(`/reviews/${review_id}/`, patchVotes)
+    .then((response) => response.data)
+    .catch((err) => console.log(err));
```
Comment (cjplatten) [inline src/api.js]: The error is not currently being returned so isn't showing on the page

**Review by cjplatten — COMMENTED** [review]
Very close, just have a look at the comment about the error

**Comment by sarah-janewilson** [comment]
Changes made

File: src/api.js (hunk 1 above)
Comment (sarah-janewilson) [reply src/api.js]: Ah of course, thank you Carrie!

File: src/Components/PostComment.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// function PostComment({review_id}) {
+//     return <p>Hi</p>
```
Comment (dappernerddesigns) [inline src/Components/PostComment.jsx:2]: Make sure to get rid of commented out code

File: src/Components/SingleReview.jsx [hunk 3]
```diff
@@ -27,15 +27,13 @@ function SingleReview() {
     };
     setCurrentReview(updatedReview);
     voteOnSingleReview(review_id, increment).catch((err) => {
-      setCurrentReview(currentReview);
-      setErrMessage("Error occurred while voting.");
-      console.err(err);
+      setErrMessage("Error occurred while voting. Your vote has not been counted. Please refresh the page and try again.");
```
Comment (dappernerddesigns) [inline src/Components/SingleReview.jsx:30]: Very informative error message, nice one.

File: src/Components/SingleReview.jsx [hunk 4]
```diff
@@ -27,15 +27,13 @@ function SingleReview() {
...
+      console.log(err);
```
Comment (dappernerddesigns) [inline src/Components/SingleReview.jsx:31]: This doesn't really need to be here however, anyone checking the console will see that the user is offline :D

**Review by dappernerddesigns — APPROVED** [review]
Looks great, some good messages to the user regarding failure. Votes persist on a refresh. Good job. 

Some code to tidy up in your api.js, you don't need catch blocks in here. Remove those and then merge away.

**Pull request merged**